### PR TITLE
[incubator/vault-ui] Initial chart

### PR DESCRIPTION
**Pull request by jpds**

### CODE REVIEW

**Comment by kachkaev** [comment]
Great to see this, thanks for crafting this chart! Just wondering if it's reasonable to mention `ingress` stuff in `README.md` so that it was clear that it's implemented.

I also suspect that mentioning k8s 1.5 only may make someone think that the chart is not compatible with k8s 1.6 and up. Maybe worth adding `+` or some other note here and in https://github.com/kubernetes/charts/pull/1397?

**Comment by andrewhowdencom** [comment]
Yo @jpds is this good to review or?

File: incubator/vault-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Vault-UI Helm Chart
+
+This directory contains a Kubernetes chart to deploy a Vault-UI server.
+
+## Prerequisites Details
+
+* Kubernetes 1.5
```
Comment (unguiculus) [inline incubator/vault-ui/README.md:7]: We support the current and the previous version, thus: 1.6+

File: incubator/vault-ui/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+```console
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install incubator/vault --set vaultUI.vault.urlDefault="http://vault:8200"
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the vault-ui chart and their default values.
+
```
Comment (unguiculus) [inline incubator/vault-ui/README.md:27]: Align table properly.

File: incubator/vault-ui/templates/_helpers.tpl [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```
Comment (unguiculus) [inline incubator/vault-ui/templates/_helpers.tpl:5]: We've started namespacing templates. See this issue and apply the same to your PR: #1785
Eventually, Helm create will do this for you in a future release. See kubernetes/helm#2845

`vault-ui.name`, `vault-ui.fullname`

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by andrewhowdencom** [comment]
@jpds  I can carry this for you if you like? I'll set a reminder to follow this up in a couple days.

**Comment by unguiculus** [comment]
Marking this as stale. Please update PR within one week.

**Comment by unguiculus** [comment]
Closing this as stale. Feel free to reopen if/when you decide to work on it again.

**Pull request closed without merging**